File: app/src/main/java/com/igdb/api_android_java/callback/onSuccessCallback.java
package com.igdb.api_android_java.callback;

import com.android.volley.VolleyError;
import com.android.volley.toolbox.Volley;

import org.json.JSONArray;
import org.json.JSONException;

/**
 * Created by captainawesome on 2017-09-12.
 */

public interface onSuccessCallback {
    void onSuccess(JSONArray result);
    void onError(VolleyError error);
}

File: app/src/main/java/com/igdb/api_android_java/callback/OnSuccessCallback.kt
package com.igdb.api_android_java.callback

import com.android.volley.VolleyError
import org.json.JSONArray

/**
 * Wrapper.OnSuccessCallback updated to Kotlin from java
 * Callback used to retrieve data from wrapper
 *
 * Created at: 2017-09-12
 * Updated at: 2018-02-10
 *
 * Created by Filip
 *
 */
interface OnSuccessCallback {
    fun onSuccess(result: JSONArray)
    fun onError(error: VolleyError)
}
